app/panel-android/components/content/FixedMenu.jsx

```diff
@@ -41,25 +42,29 @@ export default class FixedMenu extends React.Component {
 	}
 
 	get smartBlockData() {
-		return this.props.panel.smartBlock || {};
+		const { panel } = this.props;
+		return panel.smartBlock || {};
 	}
 
 	getCount = (type) => {
 		let total = 0;
 		switch (type) {
-			case 'enable_anti_tracking':
-				for (const category in this.antiTrackingData) {
-					if (this.antiTrackingData.hasOwnProperty(category)) {
-						for (const app in this.antiTrackingData[category]) {
-							if (this.antiTrackingData[category][app] === 'unsafe') {
-								total++;
-							}
+			case 'enable_anti_tracking': {
+				const categories = Object.keys(this.antiTrackingData);
+				for (let i = 0; i < categories.length; i++) {
+					const category = categories[i];
+					const apps = Object.keys(this.antiTrackingData[category]);
+					for (let j = 0; j < apps.length; j++) {
+						const app = apps[j];
+						if (this.antiTrackingData[category][app] === 'unsafe') {
+							total++;
```

Comment:
It is because the `no-restricted-syntax` linting rule throws an error when using `forEach`.
Error: `for..in loops iterate over the entire prototype chain, which is virtually never what you want. Use Object.{keys,values,entries}, and iterate over the resulting array`

Reply:
If we can use it I think it'd be nice - it's more concise and expressive

Reply:
You are right, no linting errors.  I took the "...iterate over the resulting array" in the error text to mean use a for loop but I agree, this is better.

I also updated similar loops in these files:
- app/content-scripts/click_to_play.js
- app/panel-android/components/content/FixedMenu.jsx
- app/panel/reducers/settings.js
- src/background.js
- src/classes/BugDb.js
- src/classes/Click2PlayDb.js
- src/classes/FoundBugs.js
- src/classes/PanelData.js
- src/utils/cliqzModulesData.js
- src/utils/utils.js
- tools/leet/leet-en.js

But I did __not__ update some or all of the loops in these files because `forEach` doesn't work when you need to return from within the loop, or break out of the loop.
- src/utils/matcher.js
- src/classes/BugDb.js

Changes have been made on `feature/cleanup-cleanup`, all on the `5cf04a6` commit for easy reference.

Reply:
Roger. Thank you for going through and updating everywhere it's appropriate!

Reply:
lgtm on feature/cleanup-cleanup